src/mount/sauna_client.h

```diff
@@ -148,6 +150,8 @@ struct FsInitParams {
 	             acl_cache_timeout(kDefaultAclCacheTimeout), acl_cache_size(kDefaultAclCacheSize),
 #ifdef _WIN32
 	             mounting_uid(USE_LOCAL_ID), mounting_gid(USE_LOCAL_ID),
+				 clean_acquired_files_period(kDefaultCleanAcquiredFilesPeriod), 
```

Comment:
The indentation of the added lines does not match the previous line of the block:
```suggestion
	             clean_acquired_files_period(kDefaultCleanAcquiredFilesPeriod),
```

Reply:
Done